packages/account/src/Sections/Security/LoginHistory/login-history-content.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { Table } from '@deriv/components';
+import { TLoginHistoryData } from '@deriv/api/types';
+import { fields } from '../../../../../utils/src/getLoginHistoryFormattedData';
+import LoginHistoryTableRow from './login-history-table-row';
+import LoginHistoryListRow from './login-history-list-row';
+
+type TGetFields = {
+    [key: string]: JSX.Element;
+};
+
+type TLoginHistoryContent = {
+    data: TLoginHistoryData;
+    is_mobile: boolean;
+};
+
+const getFields = () =>
```

Comment:
we can move fields to a constants file and Localize it there and directly use it here. Isnt it better? what do you think?

create a file in packages/account/src/Constants-
```
const getFields = () => ({
    date: <Localize i18n_default_text='Date and time' />,
    action: <Localize i18n_default_text='Action' />,
    browser: <Localize i18n_default_text='Browser' />,
    ip: <Localize i18n_default_text='IP address' />,
    status: <Localize i18n_default_text='Status' />,
});
```